application/frontend/controllers/EmailController.php

```diff
@@ -47,11 +51,12 @@ public function actionQueue(): Email
         }
 
         Yii::info([
-           'action' => 'email/queue',
-           'status' => 'queued',
-           'id' => $email->id,
-           'toAddress' => $email->to_address ?? '(null)',
-           'subject' => $email->subject ?? '(null)',
+            'action' => 'email/queue',
+            'status' => 'queued',
+            'id' => $email->id,
+            'toAddress' => $email->to_address ?? '(null)',
+            'subject' => $email->subject ?? '(null)',
+            'send_after' => $email->send_after,
```

Comment:
To help make the date more readable in logs:
```suggestion
            'send_after' => date('c', $email->send_after),
```